webapp/src/app/bcviewer/bcviewer.component.ts

```diff
@@ -21,6 +21,7 @@ import { UserData } from "../user-data.service";
 export class BcviewerService {
 
     @Output() update: EventEmitter<boolean> = new EventEmitter();
+    @Output() newStatus: EventEmitter<boolean> = new EventEmitter();
```

Comment:
There is only one possible value (`true` in this case), so this doesn't require a two values type, better use `void` (which contains none but the importance is that the event is triggered).
```suggestion
    @Output() newStatus: EventEmitter<void> = new EventEmitter();
```